libbpf-tools/trace_helpers.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef __TRACE_HELPER_H
+#define __TRACE_HELPER_H
```

Comment:
this is a great start, but I think this should ultimately be a small, but very useful library for working with kernel symbols and doing stack symbolization (ideally both kernel and user-space). It could be used by both BCC, as well as directly by applications using only libbpf. It's certainly not in scope for this change, just throwing the idea out. cc @brendangregg

Reply:
btw, __TRACE_HELPER(S)_H, just noticed